src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1183,6 +1183,8 @@ type QueryType {
     nearest places related to bicycling.
     """
     filterByModes: [Mode],
+    "Only include vehicle rental networks that match one of the given network names."
```

Comment:
I think the term "network name" is confusing here. Is it the network id?

Reply:
While updating it, I noticed we could also make the list items non null so I did it in the same commit https://github.com/opentripplanner/OpenTripPlanner/pull/5909/commits/87bda743e4f1672417b0b09ff6a4bc65292bcb21